eth/examples/Sushiswap/kashiBorrow.ts

```diff
@@ -0,0 +1,46 @@
+import { getContractsFactory } from '../../../src/index';
+import { BitGo } from 'bitgo';
+// Example to borrow DAI with  WETH as Collateral in Kashi
+
+async function sendBitGoTx() {
+  const bitGo = new BitGo({ env: 'test', accessToken: 'accesstoken' });
+  const baseCoin = bitGo.coin('teth');
+  const bitGoWallet = await baseCoin.wallets().get({ id: 'walletId' });
+
+  const amountWETH = '20000000000000000';
+  /*
+   * 24 ACTION_BENTO_SETAPPROVAL ParamNames user, _masterContract, approved, v, r, s. ABI encoding address, address, bool, uint8, bytes32, bytes32
+   * for information in v, r, s visit https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+   * 5 ACTION_BORROW ParamNames amount, to. ABI encoding int256, address.
+   * 20 ACTION_BENTO_DEPOSIT ParamNames token, to, amount, share. ABI encoding IERC20, address, int256, int256
+   * 10 ACTION_ADD_COLLATERAL ParamNames share, to, skim. ABI encoding int256, address, bool
+   */
+  const actions = ['24', '5', '20', '10'];
```

Comment:
unclear what values i would replace this with if i was going to copy this example, add more docs

Reply:
> unclear what values i would replace this with if i was going to copy this example, add more docs

I have added some examples, you can review now